src/config/theme.model.ts

```diff
@@ -6,6 +6,12 @@ import { getDSORoute } from '../app/app-routing-paths';
 // tslint:disable:max-classes-per-file
 export interface NamedThemeConfig extends Config {
   name: string;
+
+  /**
+   * Specify another theme to build upon: whenever a themed component is not found in the current theme,
+   * its ancestor theme(s) will be checked recursively before falling back to the default theme.
+   */
+  extends?: string;
```

Comment:
I think this `extends` is a great concept & one that people will be excited about. But I think we need to **document it**.  Could we add an example or two to `environment.common.ts`?  

I'd also really appreciate it if you could add some basic docs to https://wiki.lyrasis.org/display/DSDOC7x/User+Interface+Customization about "Extending Other Themes"(with a note saying "This feature is coming in 7.1")

Reply:
Added an example in comments.

Reply:
@ybnd : Comments look good, thanks!

If you'd like a wiki account you can request one at wikihelp@lyrasis.org.  Once you have a wiki account created, I can give you access to edit the "DSDOC7x" area of the wiki.